CHANGELOG.md

```diff
@@ -1,28 +1,72 @@
+## [15.9.0](https://github.com/commercetools/merchant-center-application-kit/compare/v15.9.0...v15.8.0) (2020-01-23)
```

Comment:
We should mention about the deprecated warnings of permissions and give a couple of examples how it's recommended to be used.